packages/studio-be/src/core/app/server.ts

```diff
@@ -145,15 +145,16 @@ export class HTTPServer {
   async getCommonEnv() {
     const config = await this.configProvider.getBotpressConfig()
 
-    return `
-    window.SEND_USAGE_STATS = ${config!.sendUsageStats};
-    window.USE_JWT_COOKIES = ${process.USE_JWT_COOKIES};
-    window.EXPERIMENTAL = ${config.experimental};
-    window.SOCKET_TRANSPORTS = ["${getSocketTransports(config).join('","')}"];
-    window.SHOW_POWERED_BY = ${!!config.showPoweredBy};
-    window.UUID = "${this.machineId}"
-    window.BP_SERVER_URL = "${process.env.BP_SERVER_URL || ''}"
-    window.IS_STANDALONE = ${process.IS_STANDALONE}`
+    return {
+      SEND_USAGE_STATS: config!.sendUsageStats,
+      USE_JWT_COOKIES: process.USE_JWT_COOKIES,
+      EXPERIMENTAL: config.experimental,
+      SOCKET_TRANSPORTS: [getSocketTransports(config).join('","')],
+      SHOW_POWERED_BY: !!config.showPoweredBy,
+      UUID: this.machineId,
+      BP_SERVER_URL: process.env.BP_SERVER_URL || '',
+      IS_STANDALONE: process.IS_STANDALONE
+    }
```

Comment:
I converted the JS function to an object as it simply makes more sense to transfer it via API than rendering it in the front-end.